gui/wxpython/datacatalog/tree.py

```diff
@@ -805,6 +844,11 @@ def InsertMapset(self, name, location_node):
         self._model.SortChildren(location_node)
         self.RefreshNode(location_node, recursive=True)
 
+    def InsertGrassDb(self, name):
+        """Insert grass db into model and refresh tree"""
+        self._model.SortChildren(self._model.root)
+        self.ReloadTreeItems()
+
```

Comment:
As I tried to explain, this should still add the node because in the future you would add a function to reload just the db here. Also add the db here to _grassdatabases. Call ReloadTreeItems from the OnAddDB handler.